pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -0,0 +1,347 @@
+           subcommand_metavar=SUBCMD_HELP_TXT)
+@add_options(help_option)
+def statistics_group():
+    """
+    Command group for WBEM server statistics.
+
+    This command group defines commands to control the display of statistical
+    data by pywbem in conjunction with the ``--timestats`` general option which
+    enables the gathering and display of statistics.
+    Since statistical information is gathered both in the pywbem client and
+    the server, there are controls for both elements.
+
+    This command group allows enabling the gathering of statistics in a
+    WBEM server by setting the CIM_ObjectManager "GatherStasticalData" property.
+
+    In addition to the command-specific options shown in this help text, the
+    general options (see 'pywbemcli --help') can also be specified before the
+    'statistics' keyword.
+    """
+    pass  # pylint: disable=unnecessary-pass
+
+
+@statistics_group.command('reset', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_reset(context):
+    """
+    Reset the values in the client local statistics.
+
+    This command does not reset statistics in the WBEM server but
+    it does reset the statistics kept in pywbemcli on the
+    ``WBEMServerResponseTime``.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_reset(context))
+
+
+@statistics_group.command('server-on', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_on(context):
+    """
+    Enable statistics on current server.
+
+    This may fail if the server does not manage statistics or does not allow
+    a client to modify the state of statistics gathering.
+
+    This does not impact pywbem local client statistics gathering. See the
+    --timestats general option.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_server_on(context))
+
+
+@statistics_group.command('server-off', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_off(context):
+    """
+    Disable statistics on current server.
+
+    This may fail if the server does not manage statistics or does not allow
+    a client to modify the state of statistics gathering.
+
+    This does not impact pywbem local client statistics gathering. See the
+    --timestats general option.
+    """
+    # pylint: disable=too-many-function-args
+
+    context.execute_cmd(lambda: cmd_statistics_server_off(context))
+
+
+@statistics_group.command('server-show', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_show(context):
+    """
+    Display statistics gathered by server.
```

Comment:
Make the short description consistent with the short description shown in the command summary starting at line 1600 of `commands.rst`.

Reply:
This is a comment on the cmd action method and not the short help.  It should provide what we need  to explain the method.